Tests/JWTKitTests/SHA256X5CTests.swift

```diff
@@ -0,0 +1,149 @@
+import Foundation
+import XCTest
+#if os(Linux)
+    import FoundationNetworking
+#endif
```

Comment:
Why this import? Unless I'm missing a usage (always possible), there's no need for it.